README.md

```diff
@@ -1 +1,113 @@
-# My-portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>Welcome to my portfolio</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] [Optional] Well! your readme file looks good overall. However, I noticed that there are section titles listed in the "Table of Contents" that do not actually exist in your readme file. This can be confusing for readers and may lead to misaligned content. kindly remove those unnecessary section titles from the "Table of Contents". 🙏 


![image](https://github.com/zhorabay/My-portfolio/assets/55840999/a9fb706e-70ee-442b-b1c0-0861616e4a01)